test/find.test.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var testHelpers = require('./test_helpers');
+
+describe('record retrival', function() {
+    var airtable;
+    var testExpressApp;
+    var teardownAsync;
+
+    beforeAll(function() {
+        return testHelpers.getMockEnvironmentAsync().then(function(env) {
+            airtable = env.airtable;
+            testExpressApp = env.testExpressApp;
+            teardownAsync = env.teardownAsync;
+        });
+    });
+
+    afterAll(function() {
+        return teardownAsync();
+    });
+
+    it('can find one record', function() {
+        var recordId = 'record1';
+
+        testExpressApp.set('handler override', function(req, res) {
+            expect(req.method).toBe('GET');
+            expect(req.url).toBe('/v0/app123/Table/record1?');
+            res.json({
+                id: req.params.recordId,
+                fields: {Name: 'Rebecca'},
+                createdTime: '2020-04-20T16:20:00.000Z',
```

Comment:
It is returned by the API so I thought I would include it to make the tests more realistic, but there aren't any methods provided by the library to access it. (Except through the `Record#_rawJson` which is clearly meant to be private. 

I could see an argument to removing it because maybe it obscures the test more by providing extraneous information. But I also see an argument of leaving it in because I imagine it might be exposed in the future and it provides a realistic understanding of the format of the json returned from the API.

Reply:
I tend toward the "extraneous information" interpretation, but I'm happy to go with your preference here, particularly because you're following precedence.